docs/tileentities/tileentity.md

```diff
@@ -0,0 +1,112 @@
+# TileEntities
+
+This is often used by vanilla Minecraft to handle inventories on chests, smelting logic on furances, or area effects for beacons. 
+More advanced versions exist in mods such as quaries, sorting machines, pipes, and displays
+!!! note
+	`TileEntities` aren't a solution for everything, their loading distance is limited, and they can cause lag when used in large amounts.
```

Comment:
This sentence is a bit messy.

AFAIK the loading distance of a tile entity is the same as its block. Did you perhaps refer to its render distance?